Resources/Prototypes/Reagents/medicine.yml

```
@@ -1589,12 +1592,6 @@
         emote: Yawn
         showInChat: true
         probability: 0.1
-      - !type:GenericStatusEffect
-        key: Drowsiness
-        component: Drowsiness
-        time: 4
-        type: Add
-        refresh: false
```

If you have an issue with sleep shot, make a new chemical, dont mess with the existing ones for this bs
```suggestion
      - !type:GenericStatusEffect
        key: Drowsiness
        component: Drowsiness
        time: 4
        type: Add
        refresh: false

---

TIL github doesnt let you webedit in lines from removed shit, fun.